journal/modernized/01-preface.adoc

```diff
@@ -374,7 +375,8 @@ the gift of grace bestowed on you through Christ Jesus,
 that your latter end may be peace, and your future state eternal happiness.
 So in Christian love remain your well-wishing friend,
 
-John Stoddart.Dublin, the 26th of the Eighth month, 1714.
```

Comment:
So, these were my best guesses at how this should be formatted, based upon what I've seen in other documents, and in the style guide. Will it be more editing stuff and trying to figure it out and make it work like this with the intakes??....I was feeling like I was changing it too much.

Reply:
Yup, this is perfect, a signature followed by a signed-section-context-close, that's exactly right.  It seems fuzzy at first, but you'll start recognizing the patterns and you'll feel more confident the more you work on this type of stuff.